WTWSMS/common/traits/WtWSMS_traits.txt

```diff
@@ -141,11 +141,14 @@ legendary_leader = {
 
 the_prophet = {
 	
-	muslim_opinion = 25
-	martial = 6
+	muslim_opinion = 100
+	martial = 12
```

Comment:
Fine on ridiculous Martial. but any opinion boost under two hundred is unrealistic. This is the founder of Islam, and all Muslims revere him unquestionably.